src/main/java/frc/robot/subsystems/lights/Lights.java

```diff
@@ -0,0 +1,78 @@
+package frc.robot.subsystems.lights;
+
+import com.ctre.phoenix.led.CANdle;
+import com.ctre.phoenix.led.CANdle.LEDStripType;
+import com.ctre.phoenix.led.CANdleConfiguration;
+import com.ctre.phoenix.led.RainbowAnimation;
+import com.ctre.phoenix.led.StrobeAnimation;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import frc.robot.Constants;
+import frc.robot.RobotMap;
+import java.util.TreeMap;
+
+public class Lights {
+  private TreeMap<LightCode, Integer[]> lightOptionsMap;
+  private CANdle candle = new CANdle(RobotMap.CANDLE_CAN_ID);
+  private CANdleConfiguration config = new CANdleConfiguration();
+  private LightCode currentLightStatus = LightCode.OFF;
+
+  public enum LightCode{
+    NOTELESS, //red
+    INTAKING, //blue
+    READY_TO_SHOOT, //green
+    NO_TAG, //purple
+    OFF, //grey
+    ALIGNING_TO_TAG, //orange
+    PARTY_MODE;
```

Comment:
I would reorder these so that OFF and PARTY_MODE are at the end. this is nitpicky though